src/adminactions/byrows_update.py

```diff
@@ -83,7 +83,10 @@ def __init__(self, *args, **kwargs):
     }
     ctx.update(modeladmin.admin_site.each_context(request))
 
-    return render_to_response(tpl, RequestContext(request, ctx))
+    if django.VERSION[:2] > (1, 8):
+        return render(request, tpl, context=ctx)
+    else:
+        return render_to_response(tpl, RequestContext(request, ctx))
```

Comment:
Instead of doing this for every single one of these views, why don't we add:

.. maybe use `LooseVersion`.. to be a little more flexible... or you can just keep `if django.VERSION[:2] > (1, 8): ..` but just dont repeat that same line 5 times.

```
from distutils.version import StrictVersion

if StrictVersion(django.__version__) > StrictVersion("1.8"):
    from django.shortcuts import render_to_response
    def render(request, template, context):
        return render_to_response(template, RequestContext(request, context))
```

add a pull request to the base repo and undo all the other code you have here.